### PR TITLE
Adding Auto update for Library Validate.js

### PR DESCRIPTION
**Pull request by abishekrsrikaanth**
Repo: https://github.com/rickharrison/validate.js
URL = http://rickharrison.github.com/validate.js/
Description = Lightweight JavaScript form validation library inspired by CodeIgniter

Additionally added the github URL on info.ini

### CODE REVIEW

File: files/validate.js/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "validate.js",
+  "repo": "rickharrison/validate.js",
+  "files": {
+    "include": ["validate.js"]
```
Comment (megawac) [inline files/validate.js/update.json]: This won't work, because the mainfile is `validate.min.js`

**Comment by megawac** [comment]
Kind of unfortunate, but because the author doesn't include a minified file I don't think this can work.

**Comment by abishekrsrikaanth** [comment]
@megawac. sent a PR to the owner. https://github.com/rickharrison/validate.js/pull/145
Hopefully it should get merged today. Shall let you know once this is ready.

**Comment by abishekrsrikaanth** [comment]
@megawac This has been added to the current master, but not part of a tagged release. How does it get deployed?

**Comment by jimaek** [comment]
A new version needs to be tagged for the bot to see the change and add the file

**Comment by abishekrsrikaanth** [comment]
@jimaek @megawac A new version of this library has been released with the minified file

**Comment by jimaek** [comment]
Yes, now please add the missing file `validate.min.js` to your `include` config.

**Comment by abishekrsrikaanth** [comment]
Sorry about that. Added the missing file to update.json

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by jimaek** [comment]
thank you

**Pull request closed without merging**